sparkmagic/sparkmagic/tests/test_livysession.py

```diff
@@ -72,10 +72,10 @@ def _next_session_response_post(self, *args):
         return val
 
     def _create_session(self, kind=constants.SESSION_KIND_SPARK, session_id=-1, sql_created=False,
-                        should_heartbeat=False):
+                        should_heartbeat=False, heartbeat_timeout=60):
         ipython_display = MagicMock()
         session = LivySession(self.http_client,
-                              {"kind": kind},
+                              {"kind": kind, "heartbeatTimeoutInSecond": heartbeat_timeout},
```

Comment:
The behavior should be that the key is added/removed based on what configuration.json dictates. You should be overriding conf (via `conf.override_all()`) instead of manually adding things here.